extensions/sd_api_pictures/script.py

```diff
@@ -29,7 +31,18 @@
     'seed': -1,
     'sampler_name': 'DDIM',
     'steps': 32,
-    'cfg_scale': 7
+    'cfg_scale': 7,
+    'characterfocus': False,
+    'nsfw': False,
+    'translations': False,
+    'character': "",
+    'anti_nsfw_prompts': '',
+    'nsfw_prompts': '(nsfw, nude, naked:1.2)',
```

Comment:
IMO better to leave this as 'nsfw' by default.

My reasoning:
- NSFW isn't strictly nude\naked, could be other things beside those two;
- attention of 1.2 is too strong when tags are scarce;
- users would be able to override it via settings.json anyway.